INDLinkLabel.swift

```diff
@@ -141,17 +149,24 @@ import UIKit
             if let URL = URL {
                 let glyphRange = self.layoutManager.glyphRangeForCharacterRange(range, actualCharacterRange: nil)
                 ranges.append(LinkRange(URL: URL, glyphRange: glyphRange))
-                
```

Comment:
Sorry about this silly whitespace change; if this is problematic, let me know and I'll resolve it.